pulp_rpm/app/tasks/copy.py

```diff
@@ -7,32 +7,6 @@
 from pulp_rpm.app.models import UpdateRecord, Package, RpmRepository, Modulemd
 
 
-def _filter_content(content, criteria, content_pks):
```

Comment:
Let's take a note of the commit or PR in the criteria issue so that we can easily bring it back later.